src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -643,6 +632,18 @@ public Void caseTemplatizedType(TemplatizedType type) {
             emit("[]");
             return null;
           }
+          // Arguments<?> and NodeList<?> in es3 externs are correspondinly
+          // IArguments and NodeList interfaces (not-parametrized) in lib.d.ts.
+          // New* are temporary work-arounds for upstream externs.
+          // TODO(rado): upgrade closure compiler and remove them.
+          if (type.getDisplayName().equals("Arguments") || type.getDisplayName().equals("NewArguments")) {
```

Comment:
can we figure out whether this is the Arguments symbol we are interested in, so we don't take other user-defined types that happen to share the name?